### ISSUE TITLE
[Bug]: [Nightly] Search string field failed caused by 'ids searched not match'

### ISSUE THREAD

**Issue by NicoYuan1986** [milvus-io/milvus#19354]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Environment

```markdown
- Milvus version:2.1.*(a0ab90ea)
- Deployment mode(standalone or cluster):cluster
- SDK version(e.g. pymilvus v2.0.0rc2):2.1.2.dev9
- OS(Ubuntu or CentOS): 
- CPU/Memory: 
- GPU: 
- Others:
```

### Current Behavior

When searching string field, error occurs.
```
[2022-09-22T00:14:55.999Z] [2022-09-21 23:56:40 - ERROR - ci_test]: list_contain_check: List([]) does not contain list([2069, 419, 893, 1293, 2706, 2041, 2004, 892, 2377, 562]) (param_check.py:99)
[2022-09-22T00:14:55.999Z] [2022-09-21 23:56:40 - ERROR - ci_test]: search_results_check: ids searched not match (func_check.py:226)
```

### Expected Behavior

PASSED

### Steps To Reproduce

```markdown
See nightly https://jenkins.milvus.io:18080/blue/organizations/jenkins/qa%2Fmilvus-release-nightly/detail/milvus-release-nightly/116/pipeline/177
```

### Milvus Log

1. collection name: search_collection_BVQYqDC0
2. FAILED timeline: [2022-09-21T23:56:40.781Z] [gw1] [ 95%] FAILED testcases/test_search.py::TestsearchString::test_search_string_field_not_primary_is_empty[False]
3. milvus log: [milvus-distributed-kafka-116-pymilvus-e2e-logs.tar.gz](https://jenkins.milvus.io:18080/job/qa/job/milvus-release-nightly/116/artifact/milvus-distributed-kafka-116-pymilvus-e2e-logs.tar.gz)

**Comment by congqixia** [comment]
Fixed by #19391

**Comment by NicoYuan1986** [comment]
The issue has been fixed.
